azext_iot/tests/iothub/devices/test_hub_export_import_int.py

```diff
@@ -45,8 +47,13 @@ def create_hub_state(self):
         labels = {generate_generic_id() : generate_generic_id(), generate_generic_id() : generate_generic_id()}
         labels = json.dumps(labels)
 
-        metrics_path = os.path.dirname(__file__) + "\\..\\configurations\\test_config_generic_metrics.json"
-        content_path = os.path.dirname(__file__) + "\\..\\configurations\\test_adm_device_content.json"
+        metrics_path = os.path.join(Path(CWD), "../configurations/test_config_generic_metrics.json")
```

Comment:
Let us make the path OS agnostic by doing the join like:
 _**os.path.join(Path(CWD).parent, "configurations", "test_config_generic_metrics.json")**_